code/game/gamemodes/wizard/raginmages.dm

```diff
@@ -63,12 +51,12 @@
 					to_chat(wizard.current, "<span class='warning'><font size='4'>The Space Wizard Federation is upset with your performance and have terminated your employment.</font></span>")
 					wizard.current.dust() // *REAL* ACTION!! *REAL* DRAMA!! *REAL* BLOODSHED!!
 			continue
-		if(wizard.current.client && wizard.current.client.is_afk() > 10 * 60 * 10) // 10 minutes
+		if(!wizard.current.client)
+			continue // Could just be a bad connection, so SSD wiz's shouldn't be gibbed over it, but they're not "alive" either
+		if(wizard.current.client && wizard.current.client.is_afk() > 10 MINUTES)
```

Comment:
Checking for the wizard client again is necessary since it will ```continue``` if one is not detected?